packages/mobile/test/schemas.ts

```diff
@@ -832,6 +832,10 @@ export const v20Schema = {
     multiTokenShowHomeBalances: false,
     multiTokenUseSendFlow: false,
     multiTokenUseUpdatedFeed: false,
+    allowOtaTranslations: false,
+    otaTranslationsLastUpdate: 0,
+    otaTranslationsAppVersion: '0',
+    otaTranslationsLanguage: '',
```

Comment:
We should create a new schema + migration (empty migration is fine here because the app reducer handles the REHYDRATE action manually and uses the reducers initial state) for these new props.

Reply:
i've incremented the redux version and added a migration :)